api/app/resources/bookings/exam/exam_list.py

```diff
@@ -32,10 +33,24 @@ def get(self):
         try:
             csr = CSR.find_by_username(g.jwt_oidc_token_info['preferred_username'])
 
+            ninety_day_filter = datetime.now() - timedelta(days=90)
+
             if csr.liaison_designate == 1:
-                exams = Exam.query.filter(Exam.deleted_date.is_(None)).all()
+                exams = Exam.query.filter(Exam.deleted_date.is_(None))\
```

Comment:
These two queries should be updated to use an `_or` filter.

https://stackoverflow.com/questions/7942547/using-or-in-sqlalchemy

You may also find it cleaner to do:

> exams = [exam_query]
> if csr.liaison_designate != 1:
>   exams = exams.filter_by(office_id=csr.office_id)

Reply:
Fixed and pushed.